src/main/java/org/folio/bulkops/service/ErrorService.java

```diff
@@ -125,7 +125,9 @@ public void saveErrorsFromDataImport(UUID bulkOperationId, UUID dataImportJobId)
         if (errorEntry.getSourceRecordActionStatus() == JobLogEntry.ActionStatus.DISCARDED && errorEntry.getError().isEmpty()) {
           errorEntry.setError(DATA_IMPORT_ERROR_DISCARDED);
         }
-        saveError(bulkOperationId, identifier, errorEntry.getError());
+        if (!errorEntry.getError().isEmpty()) {
```

Comment:
Hi @obozhko-folio , it is better to to use StringUtils to exclude null point exception

Reply:
There is a check for null on line 114:
.filter(entry -> nonNull(entry.getError()))